Loop/Managers/DoseMath.swift

```diff
@@ -83,16 +84,16 @@ extension InsulinCorrection {
     /// - Parameters:
     ///   - pendingInsulin: The number of units expected to be delivered, but not yet reflected in the correction
     ///   - maxBolus: The maximum allowable bolus value in units
-    ///   - minimumProgrammableIncrementPerUnit: The smallest fraction of a unit supported in bolus delivery
+    ///   - volumeRounder: The smallest fraction of a unit supported in bolus delivery
     /// - Returns: A bolus recommendation
     fileprivate func asBolus(
         pendingInsulin: Double,
         maxBolus: Double,
-        minimumProgrammableIncrementPerUnit: Double
+        volumeRounder: ((Double) -> Double)?
```

Comment:
[nit] `volumeRounder` to `basalVolumeRounder` for consistency.

Reply:
Same as previous.

Reply:
👍